tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES2018",
+    "target": "ES2020",
```

Comment:
This is potentially dangerous :warning:  
We want to support Node 12

Reply:
That's fine, the latest previous LTS version works flawlessly with that target.

Reply:
According to this, we should at least use es2019

https://stackoverflow.com/a/59787575/6897682

Otherwise please share your source

Reply:
Also this site https://node.green/#ES2020 says that node 12.10.0 doesn't support most of the es2020 features  
Maybe our transpiled source code isn't affected anyways, but just keep save